frontend/app/pages/dashboards/dashboard-list.js

```diff
@@ -0,0 +1,100 @@
+import template from './dashboard-list.html';
+import {_} from 'underscore';
+
+function DashboardListCtrl($scope, Dashboard, $location, currentUser, clientConfig, NgTableParams) {
+  const self = this;
+
+  this.logoUrl = clientConfig.logoUrl;
+  const page = parseInt($location.search().page || 1, 10);
+  const count = 25;
+
+  this.defaultOptions = {};
+  this.dashboards = Dashboard.query({}); // shared promise
+
+  $scope.selectedTags = []; // in scope because it needs to be accessed inside a table refresh
+  $scope.searchText = "";
+
+  $scope.$watch(function(){
```

Comment:
Why do you need this watch? You already call `reload` with `ng-change`.

Also it might be better to have an explicit search button.